src/Pages/Main/MainSlider.scss

```
@@ -0,0 +1,34 @@
+@import '../../styles/common.scss';
+
+.MainSlider {
+  width: 100%;
+  // height: 600px;
+
+  .slick-slider button{
+    width: 70px;
+    height: 70px;
+    background: rgba(255,255,255,.05);
+    z-index: 10;
+    transition: all .2s ease;
+    &:hover{
+      background: rgba(255,255,255,.2);
+    }
+    &:first-of-type {
+      left: 180px;
+    }
+    &:last-of-type {
+      right: 180px;
+    }
+    &.slick-prev::before {
+      font-size: 40px;
+      // display: none;
+    }
+    &.slick-next::before {
+      font-size: 40px;
+      // display: none;
+    }
```

* 같은 속성이라면 하나로 합쳐주세요
* 주석부분은 사용하지 않는다면 항상 삭제해주세요 
```suggestion
    &:first-of-type {
      left: 180px;
    }
    &:last-of-type {
      right: 180px;
    }
    &.slick-prev::before,  &.slick-next::before {
      font-size: 40px;
    }

```